CHANGELOG.md

```diff
@@ -88,9 +88,23 @@ Adding a new version? You'll need three changes:
  - [0.0.5](#005)
  - [0.0.4 and prior](#004-and-prior)
 
-## Unreleased
+## 3.2
 
-> Release date: TBA
+> Release date: 2024-06-12
+
+### Highlights
+
+- 🚀 **Fallback Configuration**: New `FallbackConfiguration` feature enables isolating configuration failure domains so
+  that one broken object does no longer prevent the entire configuration from being applied. See [Fallback Configuration guide]
```

Comment:
```suggestion
  that one broken objectno longer prevents the entire configuration from being applied. See [Fallback Configuration guide]
```

btw. should we emphasize here that it's alpha feature?

Reply:
@mheap What do you think?